x-pack/plugins/fleet/server/services/epm/archive/validation/validate.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import fs from 'fs';
+import path from 'path';
+
+// Provides global.Go runtime
+import './wasm_exec';
+// @ts-expect-error
+const go = new Go();
```

Comment:
__question:__ I'm not familiar with `wasm_exec.js`\tinygo, is it possible to use its `importObject` without polluting the global scope with `Go`?

Reply:
Without editing the `wasm_exec.js` from Go (this is not from tinygo), I don't see a way. I think it would be a small edit though and I agree we shouldn't merge this in its current state (polluting the global scope).